From f6316683b1024146b6e7b7d8a46c4d7f5e2dfe4c Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Tue, 10 Apr 2018 10:14:12 -0700
Subject: [PATCH 2/2] qcom/sepol: Allow mm-qcamerad to use binder even in
 vendor

Devices with this on vendor and full treble need to be able to access
binder for mm-qcamera-daemon. Also set binder_in_vendor_violator to actually
allow it to use binder.

Change-Id: I623fa5aeceafc942fb2ed9991e152505becb5139
---
 common/mm-qcamerad.te | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/common/mm-qcamerad.te b/common/mm-qcamerad.te
index 5213494..0490be9 100644
--- a/common/mm-qcamerad.te
+++ b/common/mm-qcamerad.te
@@ -1,4 +1,4 @@
-type mm-qcamerad, domain;
+type mm-qcamerad, domain, binder_in_vendor_violators;
 type mm-qcamerad_exec, exec_type, vendor_file_type, file_type;
 init_daemon_domain(mm-qcamerad)
 
@@ -27,7 +27,7 @@ unix_socket_connect(mm-qcamerad, sensors, sensors)
 #Allow connections between sensor manager and mm-qcamerad
 #allow mm-qcamerad system_server:unix_stream_socket rw_socket_perms;
 binder_call(mm-qcamerad, system_server);
-#binder_use(mm-qcamerad);
+binder_use(mm-qcamerad);
 
 allow mm-qcamerad self:socket create_socket_perms_no_ioctl;
 allow mm-qcamerad persist_file:dir r_dir_perms;
-- 
2.17.0

